check/check.go

```diff
@@ -133,6 +133,16 @@ func (c *Checker) ProgramSSA(prog *ssa.Program) {
 	c.prog = prog
 }
 
+// Algo supplies Checker with the call graph construction algorithm.
+func (c *Checker) Algo(algo string) {
```

Comment:
Use a better name for this one, like `CallgraphAlgorithm`.

Reply:
fixed